mta.yaml

```diff
@@ -186,8 +187,8 @@ resources:
     parameters:
       service: hana
       service-plan: schema
-      #config:
-        #database_id: <database_guid>
+      config:
```

Comment:
This shouldn't be required if there is only 1 default `hana-cloud` ?

Reply:
Included in the documentation ? If not, Please include the steps to fill this `<database_guid>` in the deployment guide
https://github.com/SAP-samples/cloud-espm-cloud-native#deploy-service